packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx

```diff
@@ -0,0 +1,151 @@
+import { Text } from '@saladtechnologies/garden-components'
+import classNames from 'classnames'
+import type CSS from 'csstype'
+import type { FunctionComponent } from 'react'
+import type { WithStyles } from 'react-jss'
+import withStyles from 'react-jss'
+import type { SaladTheme } from '../../../../SaladTheme'
+import { demandPillColors } from './constants'
+import { mockedDemandMonitorData } from './mocks'
+
+const styles: (theme: SaladTheme) => Record<string, CSS.Properties> = (theme: SaladTheme) => ({
+  tableWrapper: {
+    fontFamily: 'Mallory',
+    color: theme.lightGreen,
+    fontSize: '16px',
+    width: '100%',
+    '@media (max-width: 900px)': {
+      overflow: 'scroll',
+    },
+  },
+  tableContent: {
+    overflow: 'hidden',
+    borderRadius: '6px',
+    border: `1px ${theme.green} solid`,
+    minWidth: '900px',
+    '@media (max-width: 900px)': {
+      minWidth: '1000px',
+    },
+  },
+  table: {
+    width: '100%',
+    borderRadius: '6px',
+    borderCollapse: 'collapse',
+    boxSizing: 'border-box',
+  },
+  greenTableCell: {
+    backgroundColor: theme.green,
+    color: theme.darkBlue,
+  },
+  tableCell: {
+    border: `1px ${theme.green} solid`,
+    borderCollapse: 'collapse',
+    padding: '10px 24px',
+  },
+  gpuWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'column',
+  },
+  gpuName: {
+    marginBottom: '8px',
+  },
```

Comment:
```suggestion
  gpuWrapper: {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    flexDirection: 'column',
    borderTop: `1px ${theme.darkBlue} solid`,
  },
  gpuName: {
    fontWeight: 700,
    marginBottom: '8px',
  },
```

<img width="925" alt="image" src="https://github.com/user-attachments/assets/317daad0-7655-4f9d-94ea-c3a22b305102">

Reply:
Just a small suggestion to design